docs/collaboration/working_groups.md

```diff
@@ -0,0 +1,20 @@
+# BIDS Working groups
+
+BIDS working groups are typically formed (but not limited to)
+for the purpose of advancing the BIDS community, not the standard.
+
+This could be a to set up and organize a local BIDS chapter, to improve the documentation around BIDS...
+
+## Forming a working group
```

Comment:
@ericearl 
You commented that the procedure to start a working group was unclear #375, I would suggest that until we update the governance on this we improve things in this section of the website.